libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +81,46 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * Updates customer object with incoming tax ID if existing tax ID does not match
+   *
+   * @param customerId Customer ID of customer to be updated
+   * @param taxId Customer tax ID to be updated if different from existing tax ID
+   * @returns Updated customer, or undefined if existing customer tax ID matches
```

Comment:
[non-blocking] I think this should just return `true/false`, where `true` is updated, `false` is not updated (and a thrown error indicates a failure). This matches the pattern we use in our repositories.